config/initializers/will_paginate.rb

```diff
@@ -1 +1 @@
-WillPaginate.per_page = 20
```

Comment:
I tried fixing that before I committed, it appears I added a new line that I've since removed within this PR. I'm hoping that fixes this particular diff.